
From: Mikael Pettersson <mikpe@csd.uu.se>

On x86, PERFCTR_INTERRUPT_SUPPORT is supposed to be a derived
non-user-controllable option which is set if and only if X86_LOCAL_APIC is
set.  However, I broke that logic when I added preliminary user-selectable
interrupt support to ppc32.  The patch below fixes it.

Signed-off-by: Mikael Pettersson <mikpe@csd.uu.se>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/perfctr/Kconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN drivers/perfctr/Kconfig~perfctr-interrupt-support-kconfig-fix drivers/perfctr/Kconfig
--- 25/drivers/perfctr/Kconfig~perfctr-interrupt-support-kconfig-fix	2004-10-10 23:37:39.660668832 -0700
+++ 25-akpm/drivers/perfctr/Kconfig	2004-10-10 23:37:39.663668376 -0700
@@ -51,7 +51,8 @@ config PERFCTR_VIRTUAL
 	  Say Y.
 
 config PERFCTR_INTERRUPT_SUPPORT
-	bool "Performance counter overflow interrupt support"
+	prompt "Performance counter overflow interrupt support" if PPC
+	bool
 	depends on PERFCTR
 	default y if X86_LOCAL_APIC
 
_
